calico-cloud_versioned_docs/version-18-2/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -82,8 +82,7 @@ You can include the CLI scanner in your CI/CD pipelines (for example, Jenkins, G
 - Download the CLI scanner binary onto your CI runner
 - If you are running an ephemeral environment in the pipeline, include the download, and update the executable steps in your pipeline to download the scanner on every execution
 - Create a secret containing the API-Token and API URL and make it available in the pipeline (for example, using a SECURE_API_TOKEN environment variable)
-- Add a step in your pipeline to run the `image-assurance-scanner` after building the container image, and specify the image name as a parameter. For example:
-        `./image-assurance-cli-scanner --apiurl <sysdig-api-url> ${IMAGE_NAME}`
+- Add a step in your pipeline to run the `image-assurance-scanner` after building the container image, and specify the image name as a parameter. For example: `./tigera-scanner scan --apiurl <API-URL> --token $(SecureApiToken) ${IMAGE_NAME}:$(tag)`
```

Comment:
I wonder whether we would be better served without mentioning the `image-assurance-scanner` using code formatting. I think it creates a bit of confusion with `tigera-scanner` because it raises the question, which is which?

Reply:
Do we mention "image-assurance-scanner" prominently anywhere else in the docs? 

I think we could just say "Image Assurance scanner" here. The snake-case naming format implies a specific thing for developers, i.e. that it's a command-line binary called "image-assurance-scanner".

Reply:
I find the syntax for the command a bit inconsistent. 

E.g. why do we use "<...>" for "<API-URL>" but then use "${...}" for the others? 

Also, why do we have "API-URL" vs "SecureApiToken" vs "IMAGE_NAME" vs "tag"? Those are all slightly different formats. 

I don't think you can answer these questions, since it comes from the suggestion from Faisal. But we should change it to be consistent I think.

Reply:
What about this? I think an example would be helpful to add to the syntax. 

`./tigera-scanner scan --apiurl <API_URL> --token $<Secure_Api_Token> $<image_name>:$<tag>`